src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -222,7 +223,8 @@
                            ssl-config)
                         :client-auth :none
                         :cipher-suites (or (:cipher-suites ssl-config) config/acceptable-ciphers)
-                        :protocols     (or (:protocols ssl-config) config/default-protocols)}))
+                        :protocols     (or (:protocols ssl-config) config/default-protocols)
+                        :cache-session-id (get ssl-config :cache-session-id true)}))
```

Comment:
I think I would be fine, personally, with not touching this stuff for the proxy servlet for now, and addressing that in a follow-up ticket if we think it's necessary in the future.  I don't think the proxy stuff would come into play at all for the use cases where CS has run into session issues so far.